apps/extension/src/contexts/tx-context.tsx

```diff
@@ -176,56 +176,22 @@ export const TxProvider = ({ children }: { children: React.ReactNode }) => {
     userOpRef.current = null;
   };
 
-  const onSendSuccess = async (
-    opHash: string,
-    currentUserOp: ElytroUserOperation,
-    txHash?: string
-  ) => {
-    wallet.addNewHistory({
-      type: txTypeRef.current!,
-      opHash,
-      txHash,
-      userOp: currentUserOp!,
-      decodedDetail: decodedDetail!,
-    });
-    toast({ title: 'Transaction sent successfully' });
-    await resolve(txHash);
-    handleBack();
-  };
-
   const onConfirm = async () => {
     try {
       setIsSending(true);
```

Comment:
same as before. we better keep the confirming logic in this place because it avoids: 1. handling copywriting in the service layer, which is inappropriate 2. introducing mismatches with `currentApproval`, ensuring the logic remains consistent and reliable. also we need this for future development.